packages/astro-embed-youtube/YouTube.astro

```diff
@@ -33,3 +33,10 @@ const posterURL = poster || `https://i.ytimg.com/vi/${videoid}/hqdefault.jpg`;
 <script>
 	import 'lite-youtube-embed';
 </script>
+
+<style is:global>
+	lite-youtube > iframe {
+		margin-top: 0 !important;
+		height: 100% !important;
```

Comment:
Thanks for trying this @vasfvitor!

If we want to be particularly safe, we could `unset: all` and then reapply the styles from lite-youtube’s CSS: https://github.com/paulirish/lite-youtube-embed/blob/master/src/lite-yt-embed.css

```suggestion
		unset: all !important;
		width: 100% !important;
    height: 100% !important;
    position: absolute !important;
    inset: 0 !important;
    border: 0 !important;
```